app/helpers/date_helper.rb

```diff
@@ -14,11 +16,22 @@ def parse_usa_import_date(date_string)
   end
 
   def parse_usa_date(date, time_string = nil)
-    date_string = (date =~ /\d{1,2}\/\d{1,2}\/\d{4}/ ? Date.strptime($&, "%m/%d/%Y") : date).to_s
-    date_string += " #{time_string}" if time_string
+    date = date.to_s.strip
+
+    # TODO: Many tests pass either a Date, a Time, or an YYYY-MM-DD formatted
```

Comment:
There were a lot of ways I tried engineering this ranging from nested conditionals to case statements, you name it. I liked this the best because there's only one line that's "wrong" and coerces incorrect inputs into what we want. If we clean up the specs, then just the one line needs to go.